### PR TITLE
Support IVS font display, irregular ideographic font display.

### PR DESCRIPTION
**Pull request by doobo**
[issues:387](https://github.com/LibrePDF/OpenPDF/issues/387),Support IVS font display, irregular ideographic font display.

### CODE REVIEW

**Comment by andreasrosdal** [comment]
Thank you for this contribution to the project. Can you please make these changes:
-Remove the changes in checkstyle.xml
-Remove unnecessary whitespace changes.
-Add JavaDoc to all new methods.
-Add unit tests which cover the use-cases.

**Comment by doobo** [comment]
> -Remove the changes in checkstyle.xml
Without it, the compilation will not pass

File: openpdf/src/main/java/com/lowagie/text/pdf/TrueTypeFont.java [hunk 1]
```diff
@@ -51,10 +51,8 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
```
Comment (mkl-public) [inline openpdf/src/main/java/com/lowagie/text/pdf/TrueTypeFont.java:57]: Don't replace the explicit `java.util.CLASS` imports by a `java.util.*` import, that is not allowed by checkstyle options.

File: openpdf/src/main/java/com/lowagie/text/pdf/TrueTypeFont.java [hunk 2]
```diff
@@ -874,6 +878,76 @@ else if (platId == 3 && platSpecId == 10) {
...
+        for(int i = 0; i < numVarSelectorRecords; ++i) {
+            byte[] input = new byte[3];
+            this.rf.read(input);
+            int selectorUnicodeValue = this.byte2int(input, 3);
+            nonDefaultUVSOffset = this.rf.readInt();
+            mappingNums = this.rf.readInt();
+            nonDefaultOffsetMap.put(selectorUnicodeValue, mappingNums);
+        }
+
+        Iterator var15 = nonDefaultOffsetMap.entrySet().iterator();
```
Comment (mkl-public) [inline openpdf/src/main/java/com/lowagie/text/pdf/TrueTypeFont.java:908]: Please use meaningful variable names, not `var15`.

**Pull request closed without merging**